spec/services/spotify_service_spec.rb

```diff
@@ -35,6 +35,19 @@
       current_song = @service.currently_playing
       expect(current_song).to eq(json)
     end
-  end
 
+    it '#top_plays' do
+      json_response = File.open('fixtures/example_top_plays.json')
+      stub_request(:get, 'https://api.spotify.com/v1/me/top/artists?limit=5&time_range=medium_term')
```

Comment:
We could have a rails_helper method to stub this, since we'll probably need it for different tests. something like stub_top_artists. Great for now though

Reply:
Good point. I agree.

Reply:
Good idea, I'm going to create an issue for this and mark this as resolved.